site-src/v1alpha2/guides/http-routing.md

```diff
@@ -33,7 +29,7 @@ Route labels and Gateway label selectors allow Routes and Gateways to be
 bound to each other by their respective owners.
```

Comment:
The paragraph above this needs to be updated to match v1alpha2 route attachment.